### DISCUSSION TITLE
Version 0.6.4

### DISCUSSION THREAD

**Discussion by IvyZX** [google/flax#2821] [General]
## What's Changed
* Improve and migrate model_surgery guide by @IvyZX in https://github.com/google/flax/pull/2668
* Add deprecation warnings and docstring pointers to flax.training.lr_schedule by @IvyZX in https://github.com/google/flax/pull/2702
* Introduce flax.io.NotFoundError to remove tensorflow dependency by @wookayin in https://github.com/google/flax/pull/2697
* wrap Module properties by @cgarciae in https://github.com/google/flax/pull/2541
* Update the checkpointing guide with Orbax migration code examples by @IvyZX in https://github.com/google/flax/pull/2688
* Include mypy in run_all_tests by @zaxtax in https://github.com/google/flax/pull/2670
* Add serialization for Partial functions. Completes #2433 by @zaxtax in https://github.com/google/flax/pull/2557
* Cleanup DenseGeneral by @cgarciae in https://github.com/google/flax/pull/2701
* Adds wave2vec2 link by @marcvanzee in https://github.com/google/flax/pull/2718
* Remove some filterwarnings by @marcvanzee in https://github.com/google/flax/pull/2715
* Adds standardize initializer by @marcvanzee in https://github.com/google/flax/pull/2717
* Bump action/setup-python Python version to 3.9 by @marcvanzee in https://github.com/google/flax/pull/2726
* Expose module.merge_param in RTD and fix docstring by @marcvanzee in https://github.com/google/flax/pull/2680
* Revert to using public github runner pool while internal issues are fixed. by @levskaya in https://github.com/google/flax/pull/2729
* Re-enable faster github CI runners by @levskaya in https://github.com/google/flax/pull/2732
* Monitor checkpoint load/save durations. by @copybara-service in https://github.com/google/flax/pull/2629
* Add Flax Dropout guide by @8bitmp3 in https://github.com/google/flax/pull/2675
* Update readthedocs build to python 3.8 by @IvyZX in https://github.com/google/flax/pull/2746
* Update Flax BatchNorm guide title by @8bitmp3 in https://github.com/google/flax/pull/2747
* Simplify _AxisRules by @cgarciae in https://github.com/google/flax/pull/2734
* Re-added serialization error checking for dicts by @chiamp in https://github.com/google/flax/pull/2691
* Guide for Flax partitioning (auto SPMD) API by @IvyZX in https://github.com/google/flax/pull/2730
* Fix LogicalNames type by @cgarciae in https://github.com/google/flax/pull/2759
* removed duplicate flip section in docs by @chiamp in https://github.com/google/flax/pull/2757
* updated tabulate documentation by @chiamp in https://github.com/google/flax/pull/2756
* add unbind by @cgarciae in https://github.com/google/flax/pull/2674
* Remove incorrect SPMD section in linen.rst by @cgarciae in https://github.com/google/flax/pull/2766
* Adds some more comments to scope.py by @marcvanzee in https://github.com/google/flax/pull/2743
* Fix GRU typo by @jheek in https://github.com/google/flax/pull/2785
* Add metrics for async checkpoint (beacon and total write duration). by @copybara-service in https://github.com/google/flax/pull/2786
* Fix indentation by @8bitmp3 in https://github.com/google/flax/pull/2787
* Link CleanRL's PPO Implementation by @vwxyzjn in https://github.com/google/flax/pull/2780
* Flax docs restructuring by @8bitmp3 in https://github.com/google/flax/pull/2788
* Add metadata helper transform for adding axis metadata. by @levskaya in https://github.com/google/flax/pull/2783
* Update Flax mission, fix some links by @8bitmp3 in https://github.com/google/flax/pull/2797
* Updated getting started doc by @chiamp in https://github.com/google/flax/pull/2698
* Added builder functions for zeros and ones initializers by @chiamp in https://github.com/google/flax/pull/2790
* Restructure Flax Examples, add sections by @8bitmp3 in https://github.com/google/flax/pull/2801
* Update how to build flax docs by @8bitmp3 in https://github.com/google/flax/pull/2740
* Updating the doc by @arcAman07 in https://github.com/google/flax/pull/2796
* added zeros_init and ones_init builder to initializers list by @chiamp in https://github.com/google/flax/pull/2806
* Implement perturb no-op behaviour by @cgarciae in https://github.com/google/flax/pull/2728
* Fix links in Flax Optax upgrading and dataset processing by @8bitmp3 in https://github.com/google/flax/pull/2798
* improve Module.init docs by @cgarciae in https://github.com/google/flax/pull/2792
* replaced zeros and ones initializers with zeros_init and ones_init builder functions by @chiamp in https://github.com/google/flax/pull/2807
* Restructure Flax guides, add sections, update copyright year by @8bitmp3 in https://github.com/google/flax/pull/2800
* Added glossary doc by @chiamp in https://github.com/google/flax/pull/2805
* Add a comment about an error. by @nouiz in https://github.com/google/flax/pull/2810
* Add transpose_kernel argument to ConvTranspose. by @mathisgerdes in https://github.com/google/flax/pull/2578
* rename ConvLSTM to ConvLSTMCell by @cgarciae in https://github.com/google/flax/pull/2767
* Add missing fields from repr by @cgarciae in https://github.com/google/flax/pull/2803
* Allow specifying method as a string by @cgarciae in https://github.com/google/flax/pull/2809
* Release 0.6.4 by @IvyZX in https://github.com/google/flax/pull/2820

## New Contributors
* @wookayin made their first contribution in https://github.com/google/flax/pull/2697
* @vwxyzjn made their first contribution in https://github.com/google/flax/pull/2780
* @arcAman07 made their first contribution in https://github.com/google/flax/pull/2796
* @nouiz made their first contribution in https://github.com/google/flax/pull/2810
* @mathisgerdes made their first contribution in https://github.com/google/flax/pull/2578

**Full Changelog**: https://github.com/google/flax/compare/v0.6.3...v0.6.4

<hr /><em>This discussion was created from the release <a href='https://github.com/google/flax/releases/tag/v0.6.4'>Version 0.6.4</a>.</em>
